riff-raff/app/views/deploy/viewDeploy.scala.html

```diff
@@ -22,7 +23,7 @@
 
     <div class="row">
         <div class="col-md-8">
-            <h2>Deploy of @record.buildName build @record.buildId@record.metaData.get("branch").map{ branch=> [@branch]} in @record.stage.name</h2>
+            <h2>Deploy of @record.buildName build @record.buildId@record.metaData.get("branch").map{ branch=> [@branch]} in @record.stage.name on @record.time.toString(DateTimeFormat.forPattern("dd/MM/yyyy"))</h2>
```

Comment:
I'd be tempted to add this as another line rather than add to the already long title.